runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -1425,9 +1425,11 @@ Java_java_security_AccessController_getAccSnapshot(JNIEnv* env, jclass jsAccessC
 	}
 	/* AccessControlContext is allocated in the same space as the thread, so no exception can occur */
 	contextObject = vmThread->threadObject;
```

Comment:
I don't think contextObject should be set and passed in the walkState. It's expected to be an AccessControlContext, not a Thread.

Reply:
JDK24 `contextObject` stays `NULL`.